app/controllers/admins/employees_controller.rb

```diff
@@ -33,6 +33,25 @@ def update
       render :edit, notice: e.message
     end
 
+    def add_available_kudos
+      if params[:number_of_additional_kudos].to_i.between? 1, 20
+        @employees = Employee.all
```

Comment:
`@employees` variable is used only once, so I think we don' need it (you can have `Employee.all.each..` )

Reply:
It was unnecessary - changed 🙂